app/scripts/views/museum/museum-controller.js

```diff
@@ -190,6 +193,16 @@
                 searchPolygon = null;
             }
         }
+
+        function requestWithSpinner(dfd) {
```

Comment:
Hmm, name of this function is slightly confusing as it doesn't actually make the request. Perhaps `attachSpinner`?